README.md

```diff
@@ -1,2 +1,48 @@
-# webpack-practice
-Practicing setting up webpack and a template for other repositories
+# To-Do App
+
+> A simple To-Do app using DOM manipulation.
+
+## Built With
+
+- HTML/CSS
+- Javascript
+- webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A web browser
+
+### Setup
+
+- Clone or download the project to your local machine
+
+- Change to the project's root directory
+
+- Make sure to checkout to the feature branch
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** What do you think if you add some command lines on how perform these steps to help users someone maybe who doesn't know how to clone a GitHub project, access the clone directory,... This will make your documentation looks smart and professional.